src/7-to-8/summary.rst

```diff
@@ -125,6 +125,33 @@ Alternatively, task outputs can be marked as :term:`optional <optional output>`.
 This supports :term:`graph branching` and it allows the scheduler to
 correctly diagnose :term:`workflow completion`.
 
+Platform Awareness
+------------------
+
+.. seealso::
+
+   :ref:`Platforms at Cylc 8.<majorchangesplatforms>`
+
+Cylc 7 was aware of individual job hosts - one selected a host using:
+``[runtime][<namespace>][remote]host``.
+
+Cylc 8 is aware of sets of host settings called
+:term:`[job] platforms <platform>`. To set a platform use
+``[runtime][<namespace>]platform``
+
+Hosts of a platform must share a file system and :term:`job runner`:
+If one host is unavailable Cylc 8 can use other hosts
+on the same platform to interact with task jobs.
+
+The same hosts can belong to different platforms, for example
+you might be able to use your local computer to launch background or slurm
+jobs.
```

Comment:
```suggestion
you might be able to use the same host to launch both background and slurm
jobs.
```
("Your local computer" might raise unhelpful questions in user's minds, since their "local computer" is not likely to be  hooked up to their HPC in that way).